prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -2296,14 +2338,38 @@ void rxr_pkt_handle_write_rta_send_completion(struct rxr_ep *ep, struct rxr_pkt_
 	rxr_cq_handle_send_completion(ep, tx_entry);
 }
 
+static int rxr_write_atomic(struct efa_mr *efa_mr, struct iovec *iov, char* data, size_t dtsize, int op, int dt)
+{
+	char host_data[iov->iov_len];
+	uint64_t device = efa_mr->peer.device.reserved;
+	int err;
+
+	/* Step 1: Copy data from device to temporary host buffer */
+	err = ofi_copy_from_hmem(FI_HMEM_CUDA, device, host_data, iov->iov_base, iov->iov_len);
```

Comment:
use `efa_mr->iface` to be general

Reply:
this was a very good catch... thank you, changed 6 places.